src/services/dashboards/all-dashboards/modules/DashboardBoardList.vue

```diff
@@ -116,24 +128,54 @@ export default defineComponent<DashboardBoardListProps>({
                     }
                 ))),
         });
+        const deleteModalState = reactive({
+            headerTitle: i18n.t('Are you sure you want to delete dashboard?'),
+            visible: false,
+            loading: false,
+            selectedId: undefined,
+        });
+
+        const handleClickDeleteDashboard = (dashboardId) => {
+            deleteModalState.selectedId = dashboardId;
+            deleteModalState.visible = true;
+        };
+
+        const handleDeleteDashboardConfirm = async () => {
+            try {
+                deleteModalState.loading = true;
+                await SpaceConnector.clientV2.dashboard[`${state.dashboardScopeType}Dashboard`].delete({
+                    [`${state.dashboardScopeType}_dashboard_id`]: deleteModalState.selectedId,
```

Comment:
nitpick: It is my personal opinion that it would be safer to give a clear value to the api request parameter key with if else statements.